site/content/docs/main/release-instructions.md

```diff
@@ -22,97 +109,17 @@ What to include in a release blog:
-
-1.  Set your GitHub token as an environment variable. `export GITHUB_TOKEN=<your token value>`
-1.  Run `/hack/release-tools/tag-release.sh` and follow the instructions for a dry-run, without pushing to GitHub.
-1.  Run `/hack/release-tools/tag-release.sh publish` and follow the instructions when everything looks good to publish to GitHub and create the release.
-
-1.  Navigate to the draft GitHub release, at https://github.com/vmware-tanzu/velero/releases.
-
-1.  If this is a patch release (e.g. `v1.2.1`), note that the full `CHANGELOG-1.2.md` contents will be included in the body of the GitHub release. You need to delete the previous releases' content (e.g. `v1.2.0`'s changelog) so that only the latest patch release's changelog shows.
-
-1.  Do a quick review for formatting. **Note:** the `goreleaser` process should detect if it's a pre-release version, and check that box in the GitHub release appropriately, but it's always worth double-checking.
-
-1.  Publish the release.
-
-1.  By now, the Docker images should have been published. Perform a smoke-test - for example:
-
-    - Download the CLI from the GitHub release
-    - Use it to install Velero into a cluster (or manually update an existing deployment to use the new images)
-    - Verify that `velero version` shows the expected output
-
-    - Run a backup/restore and ensure it works
-
-1.  (GA Only) Merge the blog post PR.
-
-1. Update Homebrew version. From a Mac, you can run `brew bump-formula-pr` to create a new PR with the updated release's information. To make sure it's the most up-to-date:
-
-    1. If you don't already have one, create a [GitHub access token for Homebrew](https://github.com/settings/tokens/new?scopes=gist,public_repo&description=Homebrew)
-
-    1. Run `export HOMEBREW_GITHUB_API_TOKEN=your_token_here` on your command line to make sure that `brew` can work on GitHub on your behalf.
-
-    1. Run `hack/brew-update.sh`. This script will download the necessary files, do the checks, and invoke the brew helper to submit the PR, which will open in your browser.
-
-1. Update Windows Chocolatey version. From a Windows computer, follow the step-by-step instructions to [create the Windows Chocolatey package for Velero CLI](https://github.com/adamrushuk/velero-choco/blob/main/README.md)
+* Plan to publish the blog post the same day as the release.
 
 ## Announce a release
 
-Once you are finished doing the release, let the rest of the world know its available by posting messages in the following places.
-
-1. Velero's Twitter account. Maintainers are encouraged to help spread the word by posting or reposting on social media.
-1. Community Slack channel.
-1. Google group message.
+Once you are finished doing the release, let the rest of the world know it's available by posting messages in the following places.
+1.  GA Only: Merge the blog post PR.
+2. Velero's Twitter account. Maintainers are encouraged to help spread the word by posting or reposting on social media.
```

Comment:
These were all using `1.` in order to let the Markdown engine generate the numbering. The trade off here is that during editing it can be hard to keep track of the points.

So I can take or leave this change. In general, I think I would prefer to have these files written for rendering on the site.